frontend/main.go

```diff
@@ -25,8 +25,19 @@ type Config struct {
 	AOLEnabled       bool   `json:"aolenabled"`
 }
 
+// Used to request databases from the global list.
+type DBOpenRequest struct {
+	// Name of the database requested:
+	DBName string
+	// Filled out on return:
+	Database goleg.Database
+	DBError error
+	SenderChannel chan DBOpenRequest
+}
```

Comment:
This makes a lot of sense, so then when I request a new database be opened I can do 'make(chan OpenAction)`. Cool.